test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +77,18 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, AddressBook addressBook, AddressBook expectedAddressBook,
+                                                                                               String expectedMessage) {
+        CommandResult result = command.execute();
+
+        // asserts the result message is correct as expected
+        assertEquals(expectedMessage, result.feedbackToUser);
```

Comment:
> Should we be comparing the CommandResult instead, rather than just one field of it?

Yes, that's actually better. I don't mind leaving it as a TODO though.